gutenberg.php

```diff
@@ -16,20 +16,144 @@
 defined( 'GUTENBERG_DEVELOPMENT_MODE' ) or define( 'GUTENBERG_DEVELOPMENT_MODE', true );
 ### END AUTO-GENERATED DEFINES
 
-gutenberg_pre_init();
+// Minimum supported version. Has to match "Requires at least" header above.
+if ( ! defined( 'GUTENBERG_MIN_WP_VERSION' ) ) {
+	define( 'GUTENBERG_MIN_WP_VERSION', '5.7' );
```

Comment:
We could auto-generate this constant. This would be similar to how `GUTENBERG_VERSION` gets injected when building the plugin zip:

https://github.com/WordPress/gutenberg/blob/d55093a139f239c11b9c0e867ce27247b7b44d98/bin/generate-gutenberg-php.php#L35-L40

Reply:
Yep, has to match the `Requires at least` from the [plugin's headers (top of this file)](https://github.com/WordPress/gutenberg/pull/35194/files#diff-88dec7baf0002c1a6210fc289c1aa78ba5b2f0b950976da04832461b40dfa9acL6). No need to type it twice :)